Makefile

```diff
@@ -0,0 +1,67 @@
+.PHONY: all clone-quickstart install-quickstart start-quickstart install-openmct-yamcs sanity-test build-example test-getopensource test-e2e clean
+
+all: clone-quickstart install-quickstart install-openmct-yamcs sanity-test build-example test-e2e
+
+start: clone-quickstart install-quickstart install-openmct-yamcs sanity-test build-example start-openmct
+
+clone-quickstart:
+	@echo "Running target: clone-quickstart"
+	@echo "Current working directory: $(shell pwd)"
```

Comment:
I _think_ you can use `$(CURDIR)` here instead of `$(shell pwd)`?

Reply:
@jvigliotta can't because it needs to run inside of the shell of the make environment